### ISSUE TITLE
Failed to parse selection resid<200 

### ISSUE THREAD

**Issue by tonigi** [Acellera/moleculekit#126]
Numeric residue selections seem not to work any more? 

```
md=Molecule("6h1f")
md.atomselect("name CA and resid<200")
```

RuntimeError: Failed to parse selection resid<200 with error Syntax error at ''<''

**Comment by stefdoerr** [comment]
Yes I don't believe I ever implemented > < <= >= etc in atomselect.
At some point we switched from VMD atomselect to my own implementation in python so it's definitely missing some features.
I guess this is a rather useful feature so I can add it in.

**Comment by stefdoerr** [comment]
Hey @tonigi . I fixed it now on main branch https://github.com/Acellera/moleculekit/commit/2f7fe7641f29906ae3fad1711e4228cea8b23e55

**Comment by stefdoerr** [comment]
I really need at some point get to simplifying the atomselect lexer/parser because I keep on adding edge cases which can be merged into simpler rules.
